apps/aevm/src/aevm_eeevm_state.erl

```diff
@@ -119,6 +124,8 @@ init(#{ env  := Env
          , environment =>
                #{ spec => Spec
                 , options => Opts }
+         , off_chain  => IsOffChain
+         , on_chain_trees  => OnChainTrees
```

Comment:
Based on code inspection, this is not used. Would it be possible to remove it? The code is a bit complex, this would ease following it.